apps/src/templates/studioHomepages/SeeMoreCourses.jsx

```diff
@@ -35,13 +44,15 @@ const SeeMoreCourses = React.createClass({
             isRtl={isRtl}
           >
             {courses.map((course, index) =>
-              <CourseCard
-                key={index}
-                name={course.name}
-                description={course.description}
-                link={course.link}
-                isRtl={isRtl}
-              />
+              <div key={index}>
+                <CourseCard
+                  name={course.name}
+                  description={course.description}
+                  link={course.link}
+                  isRtl={isRtl}
+                />
+                {(index % 2 === 0) && <div style={styles.spacer}>.</div>}
```

Comment:
Sorry, screenshots on my machine were broken yesterday :)

I think we'll switch over to using flexbox for components such as these because it can do justified layout which solves the need for these horizontal spacers.